tools/scylla-nodetool.cc

```diff
@@ -107,6 +108,23 @@ const auto app_name = "nodetool";
 
 logging::logger nlog(format("scylla-{}", app_name));
 
+std::string doc_link(std::string_view url_tail) {
+    const auto product = scylla_product();
+    const auto version = scylla_version_short();
+
+    const auto prefix = product == "scylla-enterprise" ? "enterprise" : "opensource";
+
+    std::string branch = product == "scylla-enterprise" ? "enterprise" : "master";
+    nlog.info("doc_link(): product={}, version={}", product, version);
```

Comment:
nit, can we lower the debug level to `debug()`.

Reply:
This is debug leftover, will remove.

Reply:
Done in v2.